test/e2e/framework/util.go

```diff
@@ -1247,7 +1249,7 @@ func waitTimeoutForPodNoLongerRunningInNamespace(c *client.Client, podName strin
 }
 
 func waitTimeoutForPodReadyInNamespace(c *client.Client, podName string, namespace string, timeout time.Duration) error {
-	w, err := c.Pods(namespace).Watch(api.SingleObject(api.ObjectMeta{Name: podName}))
+	w, err := c.Pods(namespace).Watch(api.SingleObject(api.ObjectMeta{Name: podName, ResourceVersion: "1"}))
```

Comment:
It's only cases where we know the resource version that we need to send it.  This will fail on large clusters because ResourceVersion will be outside the window.